syntax/ps1.vim

```diff
@@ -104,12 +104,12 @@ syn match ps1Operator /\(^\|\s\)\@<=\. \@=/
 
 " Regular Strings
 " These aren't precisely correct and could use some work
-syn region ps1String start=/"/ skip=/`"/ end=/"/ contains=@ps1StringSpecial,@Spell
-syn region ps1String start=/'/ skip=/''/ end=/'/
+syn region ps1String start=/"/ skip=/[`]"/ end=/"/ contains=@ps1StringSpecial,@Spell 
```

Comment:
I get the same syntax either way on 'VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Apr 23 2017 19:48:19)', but,

1. Why do you put everything in brackets, which is typically used for multiple classifiers?
2. Really, the only change here regarding string escaping appears to be line 108 below, i.e. a typo where the first `'` should actually be a `\``.

The classifier brackets accomplish nothing here.

Reply:
>VIM - Vi IMproved 8.1 (2018 May 18, compiled May 18 2018 18:26:56)
MS-Windows 32-bit console version

As demonstrated in the screenshots above I can reproduce the issue using the original ps1.vim syntax file with *.ps1 files.  

Interestingly, however, I cannot reproduce the problem using the following search [in vim] against a *.txt file - i.e. the search seems to work correctly. 
    
    /@"$\|^"@\|@'$\|^'@

>Why do you put everything in brackets, which is typically used for multiple classifiers?

I had to wrap each character individually to get the syntax highlighting to work correctly. 

>The classifier brackets accomplish nothing here.

Not true, at least not so in my testing.  Perhaps what I'm experiencing is an anomaly caused by something else.  Regardless, I'm not making it up... look at the screenshots.   

It's worth noting that the following also worked in the ps1.vim syntax file; either way the pattern has to be specific [in my syntax file]

    /@"\{1}

Final note, specifying $ (EOL) failed [for me] when there's a trailing space, which is, of course stylistically wrong, but still valid. 

I suppose just close/reject this PR, and I'll modify my local syntax file as needed.

Reply:
I saw your screenshots, but when I tested your changes I was not able to reproduce them. There is a subtle bug in the regex for here-strings that you did fix, but the rest did not appear to improve anything. If you want to submit a PR for the doubled single quotes to use a backtick instead, I'd be happy to merge it.